minimalTest.js

```diff
@@ -301,16 +299,36 @@ function render() {
         );
       }, "");
 
+    const prevScrollTop = view.scrollTop;
+    const isAtBottom = view.scrollTop === 0;
     view.innerHTML = messageHistoryHTML;
 
-    // Autoscroll to new message, when scrollbar is at the bottom of chatbox
-    const isScrolledToBottom =
-      view.scrollHeight - view.clientHeight <= view.scrollTop + 1;
-
-    if (isScrolledToBottom) {
-      view.scrollTop = view.scrollHeight - view.clientHeight;
```

Comment:
this was working fine, but the new logic doesn't? it used to auto scroll to the new message whenever it pops up, unless you scroll top manually. now it behave weirdly

Reply:
ok so this is not the reason of weird behavior. but with the old logic when you manually scrolled and the new message came in nothing happened, while now it scrolls down a bit